src/labkey/query/Rows.ts

```diff
@@ -710,6 +711,9 @@ export interface IQueryRequestOptions {
 
     /** Whether all of the deletes should be done in a single transaction, so they all succeed or all fail. Defaults to true. */
     transacted?: boolean
+
+    /** Can be used to override the audit behavior for the table the query is acting on. (@link LABKEY.AuditBehaviorTypes}) */
```

Comment:
nit: This `@link` isn't how typedoc links to other types. In this case I believe it would be `[[AuditBehaviorTypes]]`.

To check you can run `yarn build` which will generate the docs and then in IntelliJ right-click on the generated `docs/index.html` and select view in browser (IntelliJ will serve up the site on a local web server). Once you're in you should be able to search for `auditBehavior` and see how this link ends up looking.

Reply:
Following the pattern of other links here, I added the missing`{`.  Thanks!